File: Codeforces/Problem 4:(E - Count)/Code.java
import java.util.Scanner; 
 
public class CF_E { 
    public static void main(String[] args) { 
        Scanner input = new Scanner(System.in); 
        String str = input.nextLine(); 
        int i, sum = 0; 
        for (i = 0; i < str.length(); i++) { 
            sum = sum + str.charAt(i) - '0'; 
        } 
        System.out.println(sum); 
    } 
}

File: Codeforces/Problem 4:(E - Count)/Code.c
#include<stdio.h>
int main ()
{
    char s1[1000000];
    scanf("%s",&s1);
    int i;
    int sum=0;
    for(i=0; s1[i]!='\0'; i++)
    {
            sum=sum+(s1[i]-'0');
    }
    printf("%d\n",sum);
}